vms/platformvm/vm_regression_test.go

```diff
@@ -1123,7 +1129,14 @@ func TestValidatorSetAtCacheOverwriteRegression(t *testing.T) {
 func TestAddDelegatorTxAddBeforeRemove(t *testing.T) {
 	require := require.New(t)
 
-	validatorStartTime := banffForkTime.Add(executor.SyncBound).Add(1 * time.Second)
```

Comment:
we don't want package-scoped fork times anymore.
Default fork times must be in test setup package; if a test needs a different fork time, it should scope it.